### DISCUSSION TITLE
How to set up the width of Dropdown so that it matches the longest dropdown item or trigger?

### DISCUSSION THREAD

**Discussion by fyy623** [heroui-inc/heroui#3118] [Q&A]
Title.

I am trying to set up a dropdown menu and here is the requirement:

1. the dropdown item should appear right below the dropdown trigger.
2. dropdown trigger and its dropdown items should have exactly the same width, which is greater one between the trigger width and the longest iteam width.

Below is the current dropdown:

![image](https://github.com/nextui-org/nextui/assets/59494795/d92e1306-9fcd-40d7-a4aa-f41e540acb7e)

And I want to make it look like this:

![1716971974127](https://github.com/nextui-org/nextui/assets/59494795/6c13e2e7-dabd-48bd-9a45-3b6d06ecb6d6)

**Comment by RanitManik** [comment]
I am encountering the same problem. Did you find a solution?

**Comment by AnYiEE** [comment]
用classNames属性配合对应的插槽给菜单设置固定宽度（width）就可以了。
